web/templates/confirm_dialog/confirm_delete_detached_attachments.hbs

```diff
@@ -0,0 +1,22 @@
+<div>
+    {{#if realm_allow_edit_history}}
+        {{#tr}}
+            The following <z-link>uploaded files</z-link> are now only accessible via this message's edit history. Would you like to delete them?
+            {{#*inline "z-link"}}<a class="uploaded_files_settings_link" href="/#settings/uploaded-files">{{> @partial-block}}</a>{{/inline}}
+        {{/tr}}
+    {{else}}
+        {{#tr}}
+            The following <z-link>uploaded files</z-link> are now only accessible via this message's edit history, which is currently disabled in this organization. Would you like to delete them?
+            {{#*inline "z-link"}}<a class="uploaded_files_settings_link" href="/#settings/uploaded-files">{{> @partial-block}}</a>{{/inline}}
```

Comment:
I feel like it's too complicated to explain the message history situation when it's disabled. I think this simpler text would work:

> The following uploaded files are no longer attached to any messages. Would you like to delete them entirely?